### ISSUE TITLE
feat(a11y): Tooltip - Mouseover cannot be closed

### ISSUE THREAD

**Issue by cguhl-adesso** [Rudo-hayate/open-webui-a11y#28]
### Check Existing Issues

- [x] I have searched for all existing **open AND closed** issues and discussions for similar requests. I have found none that is comparable to my request.

### Verify Feature Scope

- [x] I have read through and understood the scope definition for feature requests in the Issues section. I believe my feature request meets the definition and belongs in the Issues section instead of the Discussions.

### Problem Description

- there are mouseover elements which are hiding other elments and cannot be closed

<img width="740" height="601" alt="Image" src="https://github.com/user-attachments/assets/76558869-53fd-448f-8a5a-43ac18f823ea" />

### Desired Solution you'd like

- close mouseover with ESC-key
